Branch-SDK/src/io/branch/referral/Defines.java

```diff
@@ -96,7 +96,18 @@
         BranchViewAction("action"),
         BranchViewNumOfUse("number_of_use"),
         BranchViewUrl("url"),
-        BranchViewHtml("html");
+        BranchViewHtml("html"),
+
+        Path("plath"),
```

Comment:
Correcting this on my local version to `path`, but noting in case you actually meant `plath`